RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py

```diff
@@ -0,0 +1,61 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
fine to have this customisation, but it should not go in production, as far as I know ... right ?

Reply:
Correct, intended for testing, not for production.